local-volume/helm/provisioner/templates/provisioner.yaml

```diff
@@ -0,0 +1,75 @@
+{{- $configMapName := .Values.configmap.configMapName }}
+{{- $provisionerNamespace := .Values.common.namespace }}
+{{- $provisionerName := .Values.daemonset.name }}
+{{- $imageFull := .Values.daemonset.image }}
+{{- $imagePullPolicy := .Values.daemonset.imagePullPolicy }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $configMapName }} 
+  namespace: {{ $provisionerNamespace }} 
+data:
+  storageClassMap: |
```

Comment:
@dhirajh added some other configs too.  Can you link those PRs here and make sure they're included in the template?

Reply:
@dhirajh Could you please point me to PR @msau42 mentioned here.

Reply:
I found it:
https://github.com/kubernetes-incubator/external-storage/pull/414/files#diff-e8566e82a3646c8c14fbcb9c3f0e2830R79

Reply:
And also this one:
https://github.com/kubernetes-incubator/external-storage/pull/480/files#diff-7ccf5ddc828f73a774d1e2d5049325fcR25

Reply:
Make sure that these two new configs can be optional

Reply:
Thanks @msau42 for sending the loink. @sbezverk Do let me know if you have any questions.